### PR TITLE
Simple doc content edits - connectors

### PR DESCRIPTION
**Pull request by mosabua**
Simple fixes from reading all the docs - this one is for all connector docs.

- comma
- avoid future usage
- spelling

Signed-off-by: Manfred Moser <mmoser@apache.org>
Signed-off-by: Manfred Moser <manfred@simpligility.com>

Follow up to https://github.com/prestosql/presto/pull/1961

### CODE REVIEW

File: presto-docs/src/main/sphinx/connector/blackhole.rst [hunk 1]
```diff
@@ -9,17 +9,17 @@ for data reading. However, it also has some other features that allow testing Pr
 in a more controlled manner. Metadata for any tables created via this connector
 is kept in memory on the coordinator and discarded when Presto restarts.
 Created tables are by default always empty, and any data written to them
-will be ignored and data reads will return no rows.
+is ignored and data reads return no rows.
 
 During table creation, a desired rows number can be specified.
-In such case, writes will behave in the same way, but reads will
-always return specified number of some constant rows.
+In such cases, writes behave in the same way, but reads
```
Comment (findepi) [inline presto-docs/src/main/sphinx/connector/blackhole.rst]: why plural? Previous has singular "During table creation"

Comment (mosabua) [reply presto-docs/src/main/sphinx/connector/blackhole.rst]: There is no singular usage of this phrase in English. You could say

In this case

or 

In such cases 

So here we might want to change to "In this case, writes..."

**Comment by mosabua** [comment]
@findepi I will rephrase when I squash the commit from the PR provided I get committer access.

**Comment by mosabua** [comment]
Replaced by https://github.com/prestosql/presto/pull/1991

**Pull request closed without merging**